### ISSUE TITLE
Telegram candy request

### ISSUE THREAD

**Issue by spylor** [NECROBOTIO/NecroBot#3832]
"/candy" could be useful in telegram for info about your 3 stage pokémon potential evolves.

**Comment by Marsinator** [comment]
how should that work exactly

list candy for each of the potential pokemon?

**Comment by spylor** [comment]
Yes. Sorted by amount of candy/name/poké#. I'm not even sure if it can be pulled as such, but I think it would be possible to pull it for the types you have in your inventory. (Knowing how much candy you have to use on pokémon you don't have isn't very useful anyway.)
Why I posted this is because I was wondering if I already had enough candies to evolve some of the pokémon (100 candy kinds) I didn't have yet in my pokédex.

**Comment by schwartzadev** [comment]
Do you still think this should be added, and if so, you would want it to list the pokemon with the most candy highest-->lowest sorted by number of candies. Is this correct?

**Comment by spylor** [comment]
I think it could be useful like that. 
Ideally it would be using pokédex numbers and would only show the first evolution stage pokémon name/candy name but I think that would be a long stretch from how this project is build.
